### PR TITLE
fix: handle missing or empty config exports [sc-0]

### PR DESCRIPTION
**Pull request by umutuzgur**
I hereby confirm that I followed the code guidelines found at [engineering guidelines](https://www.notion.so/checkly/Engineering-Guidelines-a3a165a203a04dc1a112f0e26b2f2d3f)

## Affected Components
* [x] CLI
* [ ] Create CLI
* [ ] Test
* [ ] Docs
* [ ] Examples
* [ ] Other

## Notes for the Reviewer
* Make repoUrl optional as it is not required and didn't want to add validation
* Add projectName and logicalId validations to the config

> Resolves #[498]

### CODE REVIEW

**Comment by clample** [comment]
> Resolves #[498]

I think that this could be `Resolves #498` in order to auto-link the GH issue and have it automatically close when the PR is merged.

File: packages/cli/src/constructs/project.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ export interface ProjectProps {
   /**
    * Git repository URL.
    */
-  repoUrl: string
+  repoUrl?: string
```
Comment (clample) [inline packages/cli/src/constructs/project.ts:19]: Makes sense ✅

**Comment by umutuzgur** [comment]
Hmm, another weird error...

**Review by clample — APPROVED** [review]
Nice stuff 👍  These error messages are much nicer.

**Review by clample — APPROVED** [review]

File: packages/cli/src/services/checkly-config-loader.ts [hunk 2]
```diff
@@ -74,6 +74,10 @@ function loadFile (file: string) {
   }
 }
 
+function isString (obj: any) {
+  return (Object.prototype.toString.call(obj) === '[object String]')
```
Comment (clample) [inline packages/cli/src/services/checkly-config-loader.ts:78]: Why doesn't `instanceof` work?

Comment (umutuzgur) [reply packages/cli/src/services/checkly-config-loader.ts:78]: Short answer, it is javascript. It doesn't consider const example = "string" as a member of the String class

**Pull request merged**